sycl/source/detail/scheduler/commands.cpp

```diff
@@ -30,11 +30,14 @@
 #include <string>
 #include <vector>
 
-#ifdef __GNUG__
+#if defined __has_include
```

Comment:
Not a request to change, but seems nice to have #ifdef everywhere as we use it in all other places.
```suggestion
#ifdef __has_include
```